gtoc/gtoc.go

```diff
@@ -0,0 +1,66 @@
+// Package gtoc contains the logic to convert a GOBL envelope into a CII document
+package gtoc
+
+import (
+	"fmt"
+
+	"github.com/invopop/gobl"
+	"github.com/invopop/gobl.cii/document"
+	"github.com/invopop/gobl/bill"
+)
+
+// Converter is the struct that contains the logic to convert a GOBL envelope into a CII document
+type Converter struct {
+	doc *document.Document
+}
+
+// NewConverter Builder function
+func NewConverter() *Converter {
```

Comment:
Is `NewConverter` necessary? Given that the document is stored internally, it implies "state" rather than a generic converter. Ideally you want to hide details from the user that they don't necessarily need to use.